src/SFA.DAS.ApplyService.Application.UnitTests/Handlers/OrganisationsHandlerTests/CreateOrganisationHandlerTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ApplyService.Application.Organisations.CreateOrganisation;
+using SFA.DAS.ApplyService.Domain.Entities;
+using SFA.DAS.ApplyService.Domain.Interfaces;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApplyService.Application.UnitTests.Handlers.OrganisationsHandlerTests
+{
+    [TestFixture]
+    public class CreateOrganisationHandlerTests
+    {
+        private Mock<IOrganisationRepository> _organisationRepository;
+        private Mock<IOrganisationAddressesRepository> _organisationAddressesRepository;
+        private ManageOrganisationHandler _handler;
+
+        [SetUp]
+        public void TestSetup()
+        {
+            _organisationRepository = new Mock<IOrganisationRepository>();
+            _organisationAddressesRepository = new Mock<IOrganisationAddressesRepository>();
+            _handler = new ManageOrganisationHandler(_organisationRepository.Object, _organisationAddressesRepository.Object);
+        }
+
+        [Test]
+        public async Task CreateOrganisationHandler_creates_new_Organization_if_it_not_exists()
```

Comment:
Test names to follow standard naming convention

Reply:
Done